### PR TITLE
Feature: team page backend part

### PR DESCRIPTION
**Pull request by abderrahmaneMustapha**
# Description
server side for team page

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Review by ZibanPirate — COMMENTED** [review]
Awesome work @abderrahmaneMustapha , just added two comments

**Comment by ZibanPirate** [comment]
i also changed the target branch to [`feature/team`](https://github.com/dzcode-io/dzcode.io/tree/feature/team), future PRs will target this one as well

File: api/src/team/repository.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      "projects-v2",
+      "list.json",
+    );
+    this.projects = projects !== 404 ? projects : [];
+  }
+
+  private projects: ProjectEntity[];
+
+  public async find(): Promise<GetTeamResponseDto> {
+    const projects = await Promise.all(this.projects);
```
[the remark this answers is not in the document]
Comment (abderrahmaneMustapha) [reply api/src/team/repository.ts]: done fixed

File: api/src/team/controller.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import { TeamRepository } from "./repository";
+
+@Service()
+@Controller("/Team")
+export class TeamController {
+  constructor(private readonly contributionRepository: TeamRepository) {}
+
+  @Get("/")
+  @OpenAPI({
+    summary: "Return a list of contributions for all dzcode contributors",
```
[the remark this answers is not in the document]
Comment (abderrahmaneMustapha) [reply api/src/team/controller.ts]: done fixed

**Review by ZibanPirate — APPROVED** [review]

**Pull request merged**